packages/react-component-library/src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { storiesOf } from '@storybook/react'
```

Comment:
Why is this here? Doesn't belong in this commit

Reply:
It's outdated - I accidentally included that file in that commit and then it was cherry picked over to this branch. I have since removed the offending change with an additional commit on this branch.